src/extension.ts

```diff
@@ -129,8 +139,21 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
 
             args.unshift('-c', path.join(workspacePath, psalmConfigPath));
 
+            // end of the psalm language server arguments, so we use the php cli argument separator
+            args.unshift('--');
+

```

Comment:
we use the cli argument separator to divide the psalm language server arguments and php's arguments.